### ISSUE TITLE
command-alias entries are missing from syntax/tmux.vim

### ISSUE THREAD

**Issue by atomicstack** [ericpruitt/tmux.vim#23]
As of [`tmux.git:126d364abe2e8c063efa5a888de69`](https://github.com/tmux/tmux/commit/126d364abe2e8c063efa5a888de69), the command `server-info` (and its alias `info`) is now a command-alias for `show-messages -JT`. Similarly, `split-pane`, `splitp`, `choose-window`, and `choose-session` are all defined as such:
```
$ tmux show-options -s command-alias
command-alias[0] split-pane=split-window
command-alias[1] splitp=split-window
command-alias[2] "server-info=show-messages -JT"
command-alias[3] "info=show-messages -JT"
command-alias[4] "choose-window=choose-tree -w"
command-alias[5] "choose-session=choose-tree -s"
```
It looks as if these entries are defined in [`options-table.c`](https://github.com/tmux/tmux/blob/ee3f1d25d568a425420cf14ccba6a1b2a012f7dd/options-table.c#L222).

**Comment by ericpruitt** [comment]
This should be fixed with 58be9c5890d826681d26ac5ac21d67232ba3dc4d. Thanks for reporting this and reporting it well.

**Comment by atomicstack** [comment]
Awesome, thanks for the quick fix!

**Comment by atomicstack** [comment]
I just noticed that a few of the newly added command-aliases are not ordered alphabetically; `tmux.vim` has `choose-window`, `choose-session`, and `info` on line 111, among commands beginning with the character 'n'.

Not sure if that is intentional, but it's not 100% sorted...

**Comment by ericpruitt** [comment]
It's not intentional. I apparently made a mistake in the sorting algorithm because there are other terms that are out of order, and if I run the sorting code multiple times, the lists converge on the correct order.

**Comment by ericpruitt** [comment]
Sorting has been fixed in fe3cb957b1d8f843cdb0d3082fead0e625a27eeb. Thanks again.
